tests/submodule-version-test.ts

```diff
@@ -0,0 +1,25 @@
+import { exec } from "child_process";
+
+QUnit.module("submodule typescript version", () => {
+  QUnit.test("matches installed typescript version", assert => {
+    const readYarnList: Promise<string> = new Promise((resolve, reject) => {
+      exec('yarn list --pattern typescript', (err, stdout) => {
```

Comment:
I _think_ that `require('typescript/package').version` would be simpler, what do you think?

Reply:
That would let you do something like:

```js
QUnit.test('matches installed version', function(assert) {
  let installedVersion = require('typescript/package').version;
  let testFixturesVersion = require('../../vendor/typescript/package').version;

  assert.equal(installedVersion, testFixturesVersion);
});
```

Reply:
@rwjblue oh ya that is way better. Updated